run-automation-test.md

```diff
@@ -0,0 +1,143 @@
+    ![click devices](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button (the gear symbol).
+ 
+    ![click auto](./assets/click_auto.png )
+ 
+  + On the left-hand side, you can select your preferred language, App Type, etc.
+  In this example, we use **NodeJS** as the default language of the script. Therefore, choose **NodeJS** in `Language` section and **Hybrid/Native from Apps** in `App type` section.
+
+    ![click auto](./assets/automation.png ) 
+### 2.3 Configure automation test script
+Kobiton has already provided sample scripts for automation testing, visit [here](https://github.com/kobiton/samples) for reference. 
+> Note: In this guideline, we will use the Node.js sample (samples/javascript folder) as an example.
+
+Open automation test script file in your repository or create a new one.
+
+Add necessary variables as well as `kobitonServerConfig` and `desiredCaps` in the script with ones collected in the previous step.
+
+Example:
+
+```javascript
+const username = '<YOUR_KOBITON_USERNAME>'
+const apiKey = '<YOUR_KOBITON_API_KEY>'
+const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api-test.kobiton.com',
+  auth: `${username}:${apiKey}`
+}
+var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                '<APP_URL>', 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<DEVICE_NAME>',
+  platformVersion:    '<DEVICE_VERSION>',
+  platformName:       '<DEVICE_PLATFORM_NAME>' 
+}
+```
+### 2.4 Configure TeamCity project
+Follow steps below to setup TeamCity for automation testing.
+
+>Note: In this example, we will be running Android App Testing. Therefore, we will use `android-app-test` script.
+
+1. In your TeamCity project, on the left side of the page, click **Build Steps** -> **Add build step** to set build steps.
+
+![build step](./assets/build_step.png)
+
+2. On the right side of the page, for the **Runner type** category, choose **Command Line**.
+3. Add below commands in **Custom script** section.
+
+```
+  npm install
+  npm run android-app-test
+```
+![command](./assets/cmd.png)
+
+**(Optional) Attaching environment variable to Teamcity**
++ In your project in TeamCity, on the left side of the page, click **Parameters**. Then click **Add new parameter** button. 
+
+![parameters](./assets/param.png)
+
++ Add your Kobiton username and API key.
+
+![env.username](./assets/param_name.png)
+
+![env.apikey](./assets/param_key.png)
+
++ Open your test script, replace your Kobiton username and API key with enviromental variable name in `kobitonServerConfig`.
+
+```javascript
+const username = process.env.KOBITON_USERNAME
+const apiKey = process.env.KOBITON_API_KEY
+
+var kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api.kobiton.com',
+  auth: `${username}:${apiKey}`
+}
+```
+## 3. Run automation test on Kobiton devices
```

Comment:
The information in this section is too general, as User, I cannot get the full picture information:
- The application of this method? (apply for what...)
- What are dependencies?
- Show the flow when run the automation within this method (provide image if necessary)

Reply:
@phanthaoIT I still see that your change isn't solve the problems that Hai got